oracle/service.go

```diff
@@ -208,38 +230,76 @@ func (s *oracleSvc) processSetPriceFeed(ticker, providerName string, pricePuller
 		"provider": pricePuller.ProviderName(),
 	})
 
-	ctx := context.Background()
 	symbol := pricePuller.Symbol()
```

Comment:
symbol will be empty in Stork feeds, why not use the ticker in the config? (even if one file may pull multiple tickers)

Reply:
Im ok with this, but dont really know about the idea about the symbol (I didn't coded this)

Reply:
will update this for stork